src/internal/storage/chunk/gc.go

```diff
@@ -7,17 +7,25 @@ import (
 	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
 	"github.com/pachyderm/pachyderm/v2/src/internal/log"
 	"go.uber.org/zap"
+	"golang.org/x/sync/errgroup"
 )
 
 // GarbageCollector removes unused chunks from object storage
 type GarbageCollector struct {
-	s      *Storage
-	period time.Duration
+	s            *Storage
+	period       time.Duration
+	isConcurrent bool
 }
 
 // NewGC returns a new garbage collector operating on s
-func NewGC(s *Storage, d time.Duration) *GarbageCollector {
-	return &GarbageCollector{s: s, period: d}
+func NewGC(s *Storage, d time.Duration, opts ...GCOption) *GarbageCollector {
+	gc := &GarbageCollector{s: s, period: d}
+
```

Comment:
[nitpick] we tend not to use extra new lines throughout the PFS code base. Personally I don't mind either way but we should try to stay consistent.